README.md

```diff
@@ -13,21 +13,28 @@ Takeaway Challenge
        ':..:'                ':..:'
 
  ```
+# How to Use:
+First, load in IRB using ```irb -r './lib/takeaway.rb'```
 
-Instructions
--------
+If you would like to use a new menu, add your menu to the 'Menus' directory with the following format:
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+```food,price```
 
-Task
------
+Then, you can start your order using: ```my_order = Takeaway.new('my_file_name.csv')```
```

Comment:
This didn't work for me, as you're loading in the csv here instead of a class

Reply:
You can load it with your own menu that you would need to add into the Menu directory. Otherwise, it pulls from the default menu.